kasa/cli/discover.py

```diff
@@ -76,6 +79,28 @@ async def print_discovered(dev: Device) -> None:
     return discovered
 
 
+@discover.command()
+@click.option(
+    "--redact/--no-redact",
+    default=False,
```

Comment:
Perhaps we should redact per default, not just for this but other commands, too?

Reply:
As discussed let's keep `--no-redact` as the default